register.py

```diff
@@ -53,14 +54,24 @@ def parse_args():
         help="Enable verbose output. Useful for debugging.",
     )
 
-    args = parser.parse_args()
+    parser.add_argument(
+        "--dir",
```

Comment:
```suggestion
        "--local-dir",
```

`--dir` is pretty generic, we may want to have some other directory related options later down the line!